docs/content/getting_started/runoboardingopt1.md

```diff
@@ -29,17 +29,17 @@ draft: false
 9. In Parameters, select keyword argument then select JSON. Past below json parameters with :
     ``` 
     {                   
-                        "onboard_layer": "bronze_silver",
```

Comment:
You can specify language when using triple backticks - then code will be highlighted to make it's easier to read.

like this:

```
```json
{"a":"b"}
```

Reply:
Done!